### PR TITLE
Translated some boss tips.

### PR DESCRIPTION
**Pull request by ghost**
I translated boss tips up to Varda Soma. I didn't translate Dead_TransformMechaA_004#0 as I'm not sure about the translation.

It's sorta a Google Translate translation, but I did check boss names and videos to ensure that they were correct.

Edit: Implemented @SynthSy's corrections. I also translated Dead_BossMechaA_004#0, which I missed. oops.

### CODE REVIEW

File: UI/ui_tips.csv [hunk 1]
```diff
@@ -309,11 +309,11 @@
 01_MainLobby_116#1,"""オプションで背面タッチパッドに武器切り替えや<br>回避アクションを割り当てられる。"""
 01_MainLobby_121#1,"""“PS Vita”専用ブロックでウィークリータイムアタックに挑戦すると<br>“PS Vita”専用のランキングにエントリーされる。"""
 01_MainLobby_133#1,"""クイックメニューの表示内容は、<br><pict Sign_TRIANGLE>でパターンを変更できる。"""
-Dead_BigBeastA_001#0,"""ロックベアは顔が弱点だ。ロックオンをうまく利用して攻撃しよう。"""
+Dead_BigBeastA_001#0,"""The Rockbear's face is a weak point.<br>If you lock onto the face, you can deal increased damage."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: Don't add extra fluff such as "If you lock onto the face, you can deal increased damage."

Correct usage of the sentence would be "Try to lock on to its face and attack it!"

File: UI/ui_tips.csv [hunk 2]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
 Dead_DragonSummoner_004#0,"""When a Vol Dragon is summoned, it'll shoot a blazing fireball from the air.<br>You can avoid it by moving right below the Vol Dragon."""
 Dead_DragonSummoner_005#0,"""Poison is extremely effective for the crystal surrounding the Goronzoran.<br>When poison is used, it's easy to damage the crystals."""
-Dead_DragonGladiator_001#0,"""ドラゴン・エクスが両手を高く掲げたら強力な攻撃の合図。<br>ドラゴン・エクスの側面に回りこんで回避しよう。"""
-Dead_DragonGladiator_002#0,"""ドラゴン・エクスが剣を地面に突き刺すと、地面からの攻撃が<br>発生する。走り続けることで回避することができる。"""
-Dead_DragonGladiator_003#0,"""ドラゴン・エクスの左右の盾は破壊することができる。<br>破壊すると、剣を使用した攻撃時に大きな隙が生まれる。"""
-Dead_DragonGladiator_004#0,"""ドラゴン・エクスの頭、盾、尻尾には「ポイズン」が有効だ。<br>ポイズンがかかると、その部位にダメージを与えやすくなる。"""
-Dead_TransformMechaA_001#0,"""トランマイザーは両腕で叩きつける攻撃の後、<br>弱点のコアが露出する。背後を狙って攻撃しよう。"""
-Dead_TransformMechaA_002#0,"""トランマイザーのミサイル攻撃はこちらを追いかけてくる。<br>攻撃を行わず、移動し続けて回避しよう。"""
-Dead_TransformMechaA_003#0,"""トランマイザーはミサイルによる攻撃の後、<br>弱点のコアが露出する。ミサイルを回避したら攻撃のチャンスだ。"""
+Dead_DragonGladiator_001#0,"""When Dragon Ex lifts both of their arms at the same time, it is preparing a<br>powerful attack. Run to the side of the Dragon Ex to avoid the attack."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: Dragon Ex is a singular enemy, therefore the use of "its" instead of "their" would be optimal.

File: UI/ui_tips.csv [hunk 3]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
+Dead_DragonGladiator_002#0,"""When Dragon Ex stabs their sword into the ground, a shard of light pierces from the ground. You can keep running to avoid this."""
+Dead_DragonGladiator_003#0,"""Dragon Ex's shields on its arms can be destroyed.<br>If they are destroyed, the Dragon Ex's swords won't reach as far."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: This sentence feels off. Consider rewriting this to sound better.

Comment (SynthSy) [reply UI/ui_tips.csv]: Also, the last bit is incorrect. 

> 破壊すると、剣を使用した攻撃時に大きな隙が生まれる。

This refers to creating a gap once you've broken the parts.

File: UI/ui_tips.csv [hunk 4]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
+Dead_DragonGladiator_004#0,"""Poison is extremely effective against Dragon Ex's head, shields, and tail.<br>When using poison, it's easy to do lots of damage to break these parts."""
+Dead_TransformMechaA_001#0,"""After Transmizer slams the ground with both arms, its core is revealed.<br>You can attack this weak point to deal additional damage."""
+Dead_TransformMechaA_002#0,"""Transmizer's missiles try to home in on you. If you keep moving,<br>, you can avoid the missiles."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: Consider rewriting this sentence.

File: UI/ui_tips.csv [hunk 5]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
+Dead_TransformMechaA_003#0,"""After Transmizer fires its missiles, its core is revealed.<br>You can attack this weak point to deal additional damage."""
 Dead_TransformMechaA_004#0,"""トランマイザーのマシンガン攻撃は、ジャンプしながら<br>左右に回りこんで回避しよう。"""
-Dead_BossMechaA_001#0,"""ビッグヴァーダーの甲板に上がる前に、側面の砲台を全て破壊しよう。"""
-Dead_BossMechaA_002#0,"""ビッグヴァーダーの後方にある火炎放射器は正面に立つと危険なので<br>側面から破壊するようにしよう。"""
-Dead_BossMechaA_003#0,"""ビッグヴァーダーの甲板に登った後は、後方のミサイルポッドと<br>本体肩のミサイルポッドを破壊しよう。"""
+Dead_BossMechaA_001#0,"""You should destroy the Big Vardha's guns on the sides before climbing to the deck."""
+Dead_BossMechaA_002#0,"""The flamethrowers behind Big Vardha are dangerous when attacking from the front.<br>Try attacking them from the side."""
+Dead_BossMechaA_003#0,"""The Big Vardha has missile arrays that are used to shoot you when<br>you are on the deck. You should destory them to stop the missiles."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: > ビッグヴァーダーの甲板に登った後は、後方のミサイルポッドと<br>本体肩のミサイルポッドを破壊しよう。

Talks about destroying the missile pods on the rear and sides of Vardha.

File: UI/ui_tips.csv [hunk 6]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
 Dead_BossMechaA_004#0,"""ビッグヴァーダー本体が放つ赤い電撃はジャンプでかわそう。"""
-Dead_MachineLittleBoss_001#0,"""ヴァーダーソーマはメインブースターと3つのサブブースターを使って浮遊する。<br>サブブースターを全て破壊して、空中からの攻撃を阻止しよう。"""
-Dead_MachineLittleBoss_002#0,"""ヴァーダーソーマは3つのサブブースター、両肩のミサイルポッド<br>正面の副砲、そして主砲を破壊することができる。"""
-Dead_MachineLittleBoss_003#0,"""ヴァーダーソーマのメインブースターが青白く光ったら電撃攻撃の合図。<br>急いで離れて攻撃を回避しよう。"""
-Dead_MachineLittleBoss_004#0,"""ヴァーダーソーマは主砲からビームを放ちながら1回転する。<br>攻撃範囲は広いが、足元は安全なので近づいて回避しよう。"""
+Dead_MachineLittleBoss_001#0,"""Vardha Soma has a main booster and three sub-boosters. If you destroy the sub-boosters,<br>you can stop the Vardha Soma attacking from the air."""
+Dead_MachineLittleBoss_002#0,"""A lot of Varha Soma's parts can be destroyed. Aim to damage the three sub-boosters,<br<the two shoulder missile arrays, as well as the secondary and main guns."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: This entire sentence focuses on what you can destroy. It does not tell you about the boosters, but only the front cannon and the mini turrets.

File: UI/ui_tips.csv [hunk 7]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
+Dead_MachineLittleBoss_003#0,"""When the Vardha Soma's main booster glows pale-white, it's about to create an electrical shockwave. Run away when you see this."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: 青白く, Adjective for pale, bluish white.

File: UI/ui_tips.csv [hunk 8]
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
...
+Dead_MachineLittleBoss_004#0,"""When the Vardha Soma is spinning and shooting its laser from the main gun,<br>the laser won't hit you if you stand under Vardha Soma."""
```
Comment (SynthSy) [inline UI/ui_tips.csv]: For starters, this explains that when Soma starts to spin, it makes a full rotation once. During this time, it's best to keep on your toes (or on your feet, aka move around).

While this is what most people do when fighting Soma, this is not what the tip says.

**Review by SynthSy — CHANGES_REQUESTED** [review]
Some changes need to be addressed and performed before one should merge this.

Other than that, it's ok.

File: UI/ui_tips.csv (hunk 3 above)
Comment (ghost) [reply UI/ui_tips.csv]: Which of the sentences are you referring to?

Comment (SynthSy) [reply UI/ui_tips.csv]: Click on "Files changed" at the top, and you should see what I'm referring to.

**Review by SynthSy — APPROVED** [review]

**Pull request merged**